extensions/vapid-web-push/src/setup.tsx

```diff
@@ -0,0 +1,43 @@
+import { Action, ActionPanel, Detail, getPreferenceValues, openExtensionPreferences } from "@raycast/api";
+
+export interface Preferences {
+  email: string;
+  publicKey: string;
+  privateKey: string;
+  endpoint: string;
+  p256dh: string;
+  auth: string;
+  title: string;
+  body: string;
+}
+
```

Comment:
```suggestion

```
Already a part of raycast-env.d.ts

Reply:
I think I made the usage more obvious. Let me know if I have to fix something else.